vue-app/src/graphql/queries/GetRounds.graphql

```diff
@@ -0,0 +1,5 @@
+query GetRounds {
```

Comment:
Ideally these granular queries are going to disappear in next iterations. Having one query per page/view.